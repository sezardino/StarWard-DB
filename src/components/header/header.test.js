import React from 'react';
import {shallow} from 'enzyme';

import Header from './header';

describe('Component Header', () => {
	const component = shallow(<Header />);
	it('Component rendered correctly', () => {
		expect(component.debug()).toMatchSnapshot();
	});
});
