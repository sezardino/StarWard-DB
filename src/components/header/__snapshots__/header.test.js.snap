// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component Header Component rendered correctly 1`] = `
"<div className=\\"header d-flex\\">
  <h3>
    <Link to=\\"/\\">
      Star DB
    </Link>
  </h3>
  <ul className=\\"d-flex\\">
    <li>
      <Link to=\\"/people/\\">
        People
      </Link>
    </li>
    <li>
      <Link to=\\"/planets/\\">
        Planets
      </Link>
    </li>
    <li>
      <Link to=\\"/starships/\\">
        Starships
      </Link>
    </li>
    <li>
      <Link to=\\"/secret\\">
        Secret Page
      </Link>
    </li>
    <li>
      <Link to=\\"/login\\">
        LogIn
      </Link>
    </li>
  </ul>
</div>"
`;
